client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { format } from 'date-fns';
+import { useSelector } from 'react-redux';
+import { Autocomplete } from '@material-ui/lab';
+import { Divider, Grid, TextField, Typography } from '@material-ui/core';
+
+import ContactType from 'models/ContactType';
+import ContactStatus from 'models/ContactStatus';
+import StoreStateType from 'redux/storeStateType';
+import PhoneDial from 'commons/PhoneDial/PhoneDial';
+import InteractedContact from 'models/InteractedContact';
+import InteractedContactFields from 'models/enums/InteractedContact';
+
+import useStyles from './ContactQuestioningStyles';
+
+const ContactQuestioningInfo: React.FC<Props> = (props: Props): JSX.Element => {
+    const classes = useStyles({});
+    
+    const { interactedContact, updateInteractedContact, contactStatuses } = props;
```

Comment:
I think contactStatuses  need to be declared here as a state becuase it is only used here